enterprise/backend/src/metabase_enterprise/sandbox/api/user.clj

```diff
@@ -5,26 +5,27 @@
    [compojure.core :refer [GET PUT]]
    [metabase.api.common :as api]
    [metabase.models.user :refer [User]]
-   [metabase.util.schema :as su]
-   [schema.core :as s]
+   [metabase.util.i18n :refer [deferred-tru]]
+   [metabase.util.malli :as mu]
    [toucan.db :as db]))
 
 (def ^:private UserAttributes
-  (su/with-api-error-message (s/maybe {su/NonBlankString s/Any})
-    "value must be a valid user attributes map (name -> value)"))
+  (mu/with-api-error-message
+    [:map-of
+     :keyword
+     :any]
+    (deferred-tru "value must a map of keywords to any values")))
```

Comment:
ah right I forgot about string freeze, will change it back!

about the string->keyword, the previous version is actually wrong, because we have a middleware that will keywordize the keys of the body, so it never got string in the first place.